callisto/delivery/models.py

```diff
@@ -94,7 +94,7 @@ def _unpepper(peppered_report):
 class Report(models.Model):
     """The full text of a reported incident."""
     encrypted = models.BinaryField(blank=False)
-    owner = models.ForeignKey(settings.AUTH_USER_MODEL)
+    owner = models.ForeignKey(settings.AUTH_USER_MODEL, on_delete=models.CASCADE)
```

Comment:
fair enough, but i think adding tests to demonstrate delete behavior would be a good idea given how sensitive the data is. don't think it needs to be in this PR though-- added #178 to track it